test/FixedGovLst.t.sol

```diff
@@ -177,6 +177,33 @@ contract Constructor is FixedGovLstTest {
     assertEq(fixedLst.SHARE_SCALE_FACTOR(), lst.SHARE_SCALE_FACTOR());
     assertEq(fixedLst.name(), tokenName);
     assertEq(fixedLst.symbol(), tokenSymbol);
+    assertEq(fixedLst.decimals(), 18);
+    assertEq(fixedLst.version(), "2");
+  }
+}
+
+contract DefaultDelegatee is FixedGovLstTest {
+  function test_DefaultDelegatee() public view {
```

Comment:
We need a more expressive name for this test for `scopelint` output. Maybe "Matches The Rebasing Default Delegatee Of The Rebasing Lst" or similar

Reply:
Done: [3d7a520](https://github.com/withtally/stGOV/pull/87/commits/3d7a520307fe8a12df63ece44b2671ceaec41bb5)